core/src/main/resources/error/delta-error-classes.json

```diff
@@ -657,6 +657,19 @@
     ],
     "sqlState" : "0AKDE"
   },
+  "DELTA_FEATURES_REQUIRE_MANUAL_ENABLEMENT" : {
+    "message" : [
+      "Your table schema requires the following table feature(s) that require manual enablement: <features>.",
+      "",
+      "To do this, run the following command for each of features listed above:",
+      "  ALTER TABLE table_name SET TBLPROPERTIES ('delta.feature.feature_name' = 'supported')",
```

Comment:
I think we might want to add a warning that this is currently irreversible because it is a protocol upgrade?

Reply:
Good idea. Added